### ISSUE TITLE
Functional tests fail with deptrack changes

### ISSUE THREAD

**Issue by jmafc** [perseas/Pyrseas#129]
Nearly all functional tests are failing in the deptrack branch.

```
tests/functional/test_autodoc.py F
tests/functional/test_autodoc_dir.py F
tests/functional/test_filmversions.py ...FF.
tests/functional/test_pagila.py F
```

**Comment by jmafc** [comment]
`test_autodoc.py` and `test_autodoc_dir.py` have been fixed.

`test_film_versions.py`, specifically, `test_film_version_04` and `test_film_version_05` because the deptrack changes cause a DROP COLUMN to occur ahead of a DROP CONSTRAINT which still expects the column to be present.

The `test_pagila.py` are failing because the functions `film_in_stock` and `film_not_in_stock` get created before the table `inventory` and both functions SELECT and therefore depend on that table.

**Comment by jmafc** [comment]
This has been addressed by commits cd0e569, 5014719, 22cb832 and 4b1da08.
